control/autoware_external_cmd_selector/README.md

```diff
@@ -32,3 +32,7 @@ The current mode is set via service, `remote` is remotely operated, `local` is t
 | `/external/selected/hazard_lights_cmd`                 | autoware_vehicle_msgs::msg::HazardLightsCommand   | Pass through hazard light with current mode.    |
 | `/external/selected/heartbeat`                         | TBD                                               | Pass through heartbeat with current mode.       |
 | `/external/selected/turn_indicators_cmd`               | autoware_vehicle_msgs::msg::TurnIndicatorsCommand | Pass through turn indicator with current mode.  |
+
+## Parameters
+
+{{json_to_markdown("/control/autoware_external_cmd_selector/schema/external_cmd_selector.schema.json")}}
```

Comment:
The path for this function should be the relative path from the root directory so please 

```suggestion
{{json_to_markdown("control/autoware_external_cmd_selector/schema/external_cmd_selector.schema.json")}}
```

Reply:
@batuhanbeytekin It seems like you resolved the conversation, but it seems like the issue exists. Could you remove the first `/` in the path?